### ISSUE TITLE
Password Reset throws Exception

### ISSUE THREAD

**Issue by waynegraham** [clirdlf/participatory_voting#30]
In the production environment, resetting the user password throws an exception and the user is presented with a "We're sorry, but something went wrong" error.

**Comment by waynegraham** [comment]
```
2021-05-03T18:36:34.969349+00:00 app[web.1]: [699c28a2-26f7-4d08-a151-a544588c11e0] Completed 500 Internal Server Error in 49ms (ActiveRecord: 14.3ms | Allocations: 2179)
2021-05-03T18:36:34.972424+00:00 app[web.1]: [699c28a2-26f7-4d08-a151-a544588c11e0]
2021-05-03T18:36:34.972426+00:00 app[web.1]: [699c28a2-26f7-4d08-a151-a544588c11e0] ActionView::Template::Error (Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true):
```

**Comment by waynegraham** [comment]
closed in d7edd3f
